MOxUnit/util/moxunit_util_escape_xml.m

```diff
@@ -0,0 +1,33 @@
+function str_esc=moxunit_util_escape_xml(str)
```

Comment:
Looks to me like this will be a problem when making reports through MATLAB and viewing them on Jenkins (say), but not when making them with Octave.

The problem is the escaped characters end up in the message attribute section, not the text body section, which is copied verbatum by any viewer.

Sample:
```xml
<testcase classname="my_test_of_abs.test_abs_error" name="test_abs_error" time="0.027">
  <failure message="Error using &lt;a href=&quot;matlab:matlab.internal.language.introspective.errorDocCallback(&apos;my_test_of_abs&gt;test_abs_error&apos;, &apos;/home/scott/Documents/git/matlab-ci/tests_moxunit/my_test_of_abs.m&apos;, 19)&quot; style=&quot;font-weight:bold&quot;&gt;my_test_of_abs&gt;test_abs_error&lt;/a&gt; (&lt;a href=&quot;matlab: opentoline(&apos;/home/scott/Documents/git/matlab-ci/tests_moxunit/my_test_of_abs.m&apos;,19,0)&quot;&gt;line 19&lt;/a&gt;)
elements are not equal

First input:
7

Second input:
8
">  test_abs_error:19 (/home/scott/Documents/git/matlab-ci/tests_moxunit/my_test_of_abs.m)
  run:24 (/home/scott/Documents/git/MOxUnit/MOxUnit/@MOxUnitFunctionHandleTestCase/run.m)
  run:25 (/home/scott/Documents/git/MOxUnit/MOxUnit/@MOxUnitTestSuite/run.m)
  run:25 (/home/scott/Documents/git/MOxUnit/MOxUnit/@MOxUnitTestSuite/run.m)
  moxunit_runtests:78 (/home/scott/Documents/git/MOxUnit/MOxUnit/moxunit_runtests.m)
  runAllTests:19 (/home/scott/Documents/git/matlab-ci/runAllTests.m)</failure>
</testcase>
```